src/@orb.yml

```diff
@@ -1,86 +1,8 @@
 version: 2.1
 
 description: >
-  Simplify common tasks for building and testing Node projects.
-  Source: https://github.com/circleci-public/node-orb
+  Easily install NodeJS or its package managers. Best of all, install your node_modules with caching enabled by default.
```

Comment:
Branding and registry search suggestions:

- NodeJS -> Node.js
- The words `npm` and `yarn` should be mentioned here so that this orb comes up when searched in the registry. Yarn in particular is one of the terms we can a "no results" query for right now.
- "node_modules" - > This would be typically referred to as npm packages or just packages. `node_modules` is just a directory name which Yarn on its own may or may not use.